src/src/beneficiaries/beneficiaries.service.ts

```diff
@@ -2179,7 +2210,7 @@ export class BeneficiariesService {
 							'Make Sure Your Enrollement Status is Enrolled',
 						data: {},
 					});
-				} else {
+				} else if (programDetails?.is_eligible === 'yes' && beneficiaryUser.program_beneficiaries.status === 'enrolled') {
```

Comment:
remove `if (programDetails?.is_eligible === 'yes' && beneficiaryUser.program_beneficiaries.status === 'enrolled')` condition.

add condition enrolment no exist and learner aadhaar no and learner enrolment aadhaar no is same.